internal/eea/eea.go

```diff
@@ -207,7 +207,9 @@ func (e *EEA) FlushAll(ctx context.Context) error {
 		}
 
 		msg, err := eiw.BuildMessage()
-		if err != nil {
+		if err != nil && errors.Is(err, sql.ErrNoRows) {
```

Comment:
Shouldn't this special case rather be after the call to buildEntityWrapper ?

Reply:
oops! done